src/PKSim.Infrastructure/ORM/Repositories/ParameterMetaDataRepository.cs

```diff
@@ -49,8 +45,10 @@ protected override void DoStart()
          _parameterMetaDataList.Each(parameterMetaData =>
          {
             parameterMetaData.ParentContainerPath = _flatContainerRepository.ContainerPathFrom(parameterMetaData.ContainerId).ToString();
-            parameterMetaData.ValueOrigin = _valueOriginMapper.MapFrom(_flatValueOriginRepository.FindBy(parameterMetaData.ValueOriginId));
-            parameterMetaData.ValueOrigin.Default = !parameterMetaData.IsInput;
+            //Use clone here so that we do not change the default flag 
+            var valueOrigin = _valueOriginRepository.FindBy(parameterMetaData.ValueOriginId).Clone();
```

Comment:
We want to use another instance here... not the one from db but the one specific to this parmaeterMetaData